optimum/exporters/onnx/__main__.py

```diff
@@ -444,6 +447,11 @@ def main_export(
             f" `--task {task_non_past} --monolith`, or `--task {task}` without the monolith argument."
         )
 
+    if library_name == "timm" and no_dynamic_axes is False and "vovnet" in model_type:
+        raise ValueError(
+            f"The export of {model_type} is not supported with dynamic axes. Please pass --no-dynamic-axes to export the model with fixed shapes"
+        )
```

Comment:
I wonder rather avoid adding more model-specific code in here. Is it possible to move it elsewhere (like `model_configs.py`)?

Reply:
This could not be possible as the variable (`no_dynamic_axes`) is not under the scope of model_configs.py. 

For now I have removed this from here. In documentation, it is mentioned that the following architecture support is limited.